remediation_worker/jobs/azure_security_center_enable_ddos_protection/README.md

```diff
@@ -0,0 +1,63 @@
+# Enable DDos protection for Virtual Network
+
+This job enables DDos protection for a virtual network by listing all the available DDos protection plans and assigning any one to the virtual network.
+
+### Applicable Rule
+
+##### Rule ID:
+5c8c26997a550e1fb6560cd9

```

Comment:
Check the Rule ID.
This RuleID is for `Blob container has public read access enabled`
[Secure state Link](https://www.securestate.vmware.com/rule-management/rule/5c8c26997a550e1fb6560cd9)

I think you wanted to use this [one](https://www.securestate.vmware.com/rule-management/rule/3abf3147-ea53-4302-b237-caab4d764c77)